db/install.php

```diff
@@ -0,0 +1,52 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Installation for tool_userupsert.
+ *
+ * @package     tool_userupsert
+ * @copyright   2021 Catalyst IT
+ * @license     https://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die();
+
+/**
+ * Install hook.
+ */
+function xmldb_tool_userupsert_install() {
+    // Don't set default config during tests.
```

Comment:
Why don't we set this for unit tests too? It makes default behaviour in unit tests and real life to be a bit different.

Reply:
I don't test a default behaviour with these settings. Instead, I set up each tests to have data for testing a particular test case.  Those settings like a pre populated minimum example settings which could be changed if required.